lib/simp/cli/commands/bootstrap.rb

```diff
@@ -545,8 +581,13 @@ def self.warn(message, options=nil, console_prefix='> ')
     log_and_say("WARNING: #{message}", options, console_prefix)
   end
 
-  def self.error(message, options=nil, console_prefix='> ')
+  def self.error(message, options='red.bold', console_prefix='> ')
+    log_and_say("ERROR: #{message}", options, console_prefix)
+  end
+
+  def self.fail(message, options='red.bold', console_prefix='> ')
     log_and_say("ERROR: #{message}", options, console_prefix)
+    exit 1
```

Comment:
simp cli has been carefully reworked so that no commands exit, but instead raise exceptions.  (See https://github.com/simp/rubygem-simp-cli/blob/master/lib/simp/cli.rb#L85)  This allows the command classes to be unit tested.  I realize simp bootstrap is not unit-testable at this point, but I don't want to see exit() instances cropping up in the code.

The correct solution is to define an exception to be caught by Simp::Cli::start for which the backtrace will not be logged.